src/test/java/pages/TablesPage.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class TablesPage extends BasePage {
+    final String ENDPOINT = "/tables";
+    public static final By TABLE1 = By.id("table1");
+    private String cell = "";
```

Comment:
this is not used, we can remove it (try not to keep extra code in your projects)

Reply:
I'm just haven't finished with this class=) I send it to u by mistake